services/web/server/tests/unit/with_dbs/02/common.py

```diff
@@ -0,0 +1,6 @@
+from typing import TypedDict
```

Comment:
NOTE: Tests folders must only contain `conftest.py` and `test_*.py` files. FOr the rest please put them in `pytest_simcore`. 

Experience has proben that having modules here cause import problems (e.g. tests will depend on the test workdir and you you need to add `__init__.py` everywhere) . For that reason we move all our helpers and fixtures to `pytest_simcore`.

This could perfectly go e.g. in `packages/pytest-simcore/src/pytest_simcore/helpers/typing_mock.py`